### PR TITLE
Fix login

### PR DESCRIPTION
**Pull request by bruvv**

### CODE REVIEW

**Comment by steffanjensen** [comment]
I don't like the commit title.

**Comment by ohld** [comment]
Thanks!

**Comment by ohld** [comment]
Thanks!

**Comment by steffanjensen** [comment]
I use the commit and repo search a lot to find issue for current problems. Titles like this makes searching for error fixes and problem a bit more complicated. I would recommend you to be more aware of your commit text next time. 

Because a fix login is a really broad term. A better option would be "fixed login for x example file". But thanks for your contribution :)

**Comment by ohld** [comment]
@reliefs thank you for your comment. I would recommend you to comply with the Instabot License agreement and put a link to the Instabot project in your Instabotai project sources because you use my Intellectual property. The full list of license conditions can be found in the Apache 2.0 license.

I will not contact any legal services so this action will remain on your conscience.

**Comment by bruvv** [comment]
@reliefs are you for real? I understand your point but you earning money of my work and especially @ohld and complaining about that you've to open a PR so you can merge it into your own bot is just crazy. Mind share your income from the bot with us? I might be able to listen to you.

**Comment by steffanjensen** [comment]
Sure do what you want, you first signed the License today.. Also instabot is only a small part of the code. I have not earned more than $11 last month without expenses so around $2-4, so there isn't really anything to do.. the code is too buggy now.
I really don't understand your question last time you wanted my entire script over to your repo.. Instabot is only 10% of the script.. 

Do you want me to put your License into my instabot folder ? 

I am now focusing on http://www.rankbabes.com which will release in a few days.. and the instabotai part is taken fully out of the.. If instabotai does not get fixed i will either go away from python to php, because my issues are process issues,. 

Same process each user delete each others process, because they both execute script from one file and subprocess is impossible because 2 login tries most like recreate mobile/email challenge. Also i have the api start request error.. But i have focused most on http://rankbabes.com 
If you guys want to help me to get it to run again here is the code the website runs on:
https://github.com/reliefs/InstabotAI-Linux-build
Which is part of over 10 automatic website creators i have build. 
https://github.com/users/reliefs/projects/1

This is the projects overall overview. As for now.. I still have too many bugs for instabotai.. But you are more than welcome to help me, but i have to focus on the overall goal.. and in the new script with rankbabes.com i have changed instabot to another api. 

Here are more api/bot i have build for my overall goal to my website.
https://github.com/instabotai/tiktokbot
https://github.com/instabotai/tiktok-api
https://github.com/reliefs/gitbot

I am really not out here to make issues worse, but too many issues with you guys asking me for favour new scripts, i should change this and that.. so i changed the api on my overall project.. This will most likely also happen for instabotai which is only 10% of my work.

I am pretty new to python and github, so there are many terms i don't understand, but the stuff you guys has asked me to do and some i have done, is really crazy.. You guys are great, nothing wrong there. But i have contributed a lot to the code not made any money really.. not more than $50-80 over 6 months and my expenses have been around $50 and that is not talking about getting the hardware.

So i have earned in total around $20-50 in 6 months. 

But that is "my own mistake" i didn't get any sells too many errors and being only one person contribute to the code (with 2 there contributed syntax error lol). It has been hard, but my own fauly my code is s*** and new to python.

And that was without expenses which was me running the servers. I could do a lot for your project. But as said was only 10% of my overall written code and now it's down to 2% or maybe even 0% on rankbabes. The overall code written by me is over 4000-10.000 lines and i have made so little money and even contributed stuff like SEO, I made seo for your website and fixed the css, story viewer, auto video scraper and much more. I also have AI photo for all tasks which is free and open source.

I really want to help you guys, but i don't earn enough money and always get asked about doing stuff for you guys.. what next taking my sleep patterns to fix story_viewer asking even though i have asked @ohld to please don't release stuff with same patterns as me, because then my script gets limited.. let's see.

I have told you guys about a lot of important fixes, it would take me days to do it.. 

Tldr,

- I made your website seo
- i fixed css padding and margin issues and some errors.
- I made your description tags and description be good with seo.
- I Have uploaded many examples to the github.
- Only 20% of my backend is instabot and around 1-2% of my overall code and with new update it become less, in my overall project http://www.instabotai.com is instabot completely skipped, not only because i think you have many demands.

I know there are contributes are other who have not done anything for instabot making almost millions.. So don't get wild over a $50 dollar man over 6 months and 400-600-1000 hours of work.

**Comment by ohld** [comment]
> Instabot is only 10% of the script

This doesn't matter at all. Think about it.

I know that this is my mistake - I've created a license file a couple of years ago and I thought that it would have the power I want. I've contacted GitHub support and they told that "sorry, you have to sign it first". I've trusted the GitHub's helper window that told that "Copyright notice" is one of the required License conditions. I know that legally I fucked up.

But the point is the developer ethics. There would be no problem if you forked the project, make all the fixes you want and use your forked repo as git submodule - in that case, GitHub handles all copyrighting by itself, this is very useful. But you just copypasted sources into your project, put your own license there, removed all links to original contributors like every line was written by you.

I think your future colleagues should be aware of that situation that you have appropriated the source code written by other people, even if it was legally possible by my mistake. I'm attaching all screenshots I made - I really hope that they can help to prevent this or similar behavior from the future and can save a lot of effort and time. 

Screenshot 2019-12-19 at 22 11 59
Screenshot 2020-01-06 at 15 30 27
Screenshot 2020-01-06 at 16 35 22
Screenshot 2020-01-07 at 10 15 43
Screenshot 2020-01-07 at 10 15 48

I learned a lot from this situation, so thank you - next time I will be smarter.

**Comment by steffanjensen** [comment]
Please stop cotacting me, I am not doing anything wrong.. What you are doing now isharrasment and illegal. 

Here are danii begging me to move my repo to his organisation because 10% of my code is instabot haha..
Screenshot 2020-01-06 at 15 30 27

stop harrasing me, saying i did stuff wrong while you did it.. is bad style

I would have no issues with helping others out, i have an issue because it's you danii.. 
1. I don't want to hear from you.
2. I don't want to hear your crazy demands.
3. I don't want my work under your name.

So now that both github and me have told you, you don't have a case. Can you please stop pushing this.. this has been going on for seven months now.. I really don't have time for small stuff like this.

**Comment by steffanjensen** [comment]
I also have contributed code to the repo under the old license, now because you changed license i do not agree with the new terms.. So i want all my contributions deleted..

Thanks for understanding. I will give you 48 hours to do it or i will contact Github.

**Comment by ohld** [comment]
lol

**Comment by steffanjensen** [comment]
kk i will contact github have a nice day.

**Comment by bruvv** [comment]
https://github.com/instagrambot/instabot/commit/7dfe19d92ebfa6102f05b25b5afb438759098432#commitcomment-36698234
just make a reference link so we have all your excuses everywhere to be seen

**Comment by bruvv** [comment]
https://github.com/instagrambot/instabot/commit/7dfe19d92ebfa6102f05b25b5afb438759098432
just make a reference link

**Comment by ohld** [comment]
@bruvv This is not a mistake - "instabotai" project was in "instagrambot" organization a while ago. Since this project was started and maintained by @reliefs, he decided to create his own organization and transfer the repo to it.

Thank you @bruvv for your support, I really appreciate that. But please, don't try to be rude. As for me, this looks unprofessional. I truly believe that @reliefs understand the whole situation and will do the right thing soon.

**Comment by steffanjensen** [comment]
No you are lying i created the repo months before and then you asked me to transfer it to your repo.. i did that.. No one has contributed to instabotai except me, so after i started in your organisation you wanted a lot of stuff from me and my bot got more issues so i transferred it out again.. Then you complained for 7 months now.

You have never contributed to my code that is nonsense, if you don't delete my code from your repo i will file a DMCA takedown and and copyright issue for my code in your repo.. I do not agree to the new license terms and you have to respect my copyright for my own code.. thanks

You have never written anything of instabotai that is nonsense and 90% of it all is free open source templates we both use.. so what is your issue.

For me that you now try to say you created instabotai is complete nonsense, you have not contributed one line of code except for import instabot.. out of 5000-10.000 lines i have written myself. 

Why should I give you 10.000 lines of code i have written.. you must be crazy..

**Comment by ohld** [comment]
https://github.com/instagrambot/instabot/commit/7dfe19d92ebfa6102f05b25b5afb438759098432
